demos/snackbar.html

```diff
@@ -33,6 +33,14 @@
         background-color: #333;
       }
 
+      .mdc-theme--dark .mdc-checkbox .mdc-checkbox__native-control:enabled:not(:checked):not(:indeterminate) ~ .mdc-checkbox__background {
```

Comment:
Now that #2100 is merged, we can remove these two `mdc-theme--dark` CSS rules! 😀